### DISCUSSION TITLE
Welcome to wg Discussions!

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kvark** [comment]
Hi 🦀 s!
I used to develop games professionally, and now spending nights building [wgpu](https://github.com/gfx-rs/wgpu), [naga](https://github.com/gfx-rs/naga), and related ecosystem projects. Been using Rust from dark ages, and believe we can build great stuff together with it.

**Comment by AlexEne** [comment]
Hi all!

I also used to develop games, now work @ Prime Video during the day and on https://dwarf.world/ in my weekends/evenings :D.
What I do in this community? I lurk on discord and am amazed at what cool stuff everyone is building :D
